docs/source/class_basics.rst

```diff
@@ -77,6 +77,53 @@ to it explicitly using ``self``:
            a = self
            a.x = 1      # Error: 'x' not defined
 
+Class and instance variable annotations
+***************************************
+
+Mypy supports annotations for class and instance 
+variables in class bodies and methods. Use ``ClassVar`` to
+indicate to the static type checker that this variable 
+should not be set on instances. 
+
+.. code-block:: python
+
+  class A:
+      y: ClassVar[Dict[str, int]] = {}  # class variable
+      z: int = 10                       # instance variable
+
+
+``ClassVar`` accepts only types and cannot be further subscribed.
```

Comment:
Maybe instead of having very short paragraphs introduce the relevant rules in a bullet list. It's worth mentioning that `y: ClassVar =0` is also valid (without square brackets) and the type of the variable will become implicitly `Any`. Also there should be a note that this behavior may change in the future.